migrations/20200608133617-update_list_observation_solutions_query.js

```diff
@@ -0,0 +1,35 @@
+const Uuid = require('cassandra-driver').types.Uuid;
+
+module.exports = {
+  async up(db) {
+    global.migrationMsg = "Update list observation solutions query";
+    
+    
+    if (!cassandra) {
+      throw new Error("Cassandra connection not available.");
+    }
+
+    const query = 'SELECT id FROM reports.druidqueries WHERE qid = ? ALLOW FILTERING';
```

Comment:
@deepakr123 - Take table name from env or config.json, for this and all other migrations